### ISSUE TITLE
bug in space invaders

### ISSUE THREAD

**Issue by yonigottesman** [simoninithomas/Deep_reinforcement_learning_Course#68]
the line:
# Q is our predicted Q value.
self.Q = tf.reduce_sum(tf.multiply(self.output, self.actions_))

should be:
self.Q = tf.reduce_sum(tf.multiply(self.output, self.actions_), axis=1)
